guidemaker-ember-template/src/components/search-input.hbs

```diff
@@ -1,45 +1,67 @@
-{{!-- template-lint-disable link-rel-noopener no-action no-curly-component-invocation --}}
-{{#if this.searchService.index}}
-  <input
-    id="search-input"
-    type="search"
-    value={{this.value}}
-    oninput={{perform this.search value="target.value"}}
-    placeholder="Search the guides"
-    autocomplete="off"
-    onfocus={{action "onfocus"}}
-    onblur={{action "onblur"}}
-    data-test-search-input
-  >
+<div class="search-input" ...attributes>
+  {{#if this.searchService.index}}
+    <input
+      id="search-input"
+      type="search"
+      value={{this.value}}
+      oninput={{perform this.search value="target.value"}}
+      placeholder="Search the guides"
+      autocomplete="off"
+      data-test-search-input
+      {{on "focus" this.onfocus}}
+      {{on "blur" (perform this.closeMenu)}}
```

Comment:
Is using 'perform' the standard in this project? To explicitly indicate the action? 
In the line above it's commenly written.